### PR TITLE
More EP paradigms

### PR DESCRIPTION
**Pull request by hubertjb**
This PR adds 3 new paradigms, including stimulus presentation scripts, stimuli, data for 1 subject, and analysis notebooks:

1. Auditory oddball
2. N170
3. SSVEP

I reorganized the repo a little bit so that all stimulus presentation scripts are under the same directory. I also made a utilities script under `/muse/utils.py` to keep functions that can be reused across the analysis notebooks.

Please let me know what you think! :)

### CODE REVIEW

File: muse/utils.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+import numpy as np
+import seaborn as sns
+from matplotlib import pyplot as plt
+
+
+sns.set_context('talk')
+sns.set_style('white')
+
+
+def load_muse_csv_as_raw(data_dir, subject_nb=1, session_nb=1, sfreq=256.,
```
Comment (alexandrebarachant) [inline muse/utils.py]: Can you make an extra method that takes an absolute path to a file and return the raw.

**Comment by alexandrebarachant** [comment]
Thanks, this is great.

Can you update the readme (the path for the paradigm scripts have changed)

Also, I could be great to describes shortly the different paradigms and how to run them.

**Comment by hubertjb** [comment]
I updated the README, let me know what you think. I don't know if a table is really useful :P

**Comment by alexandrebarachant** [comment]
Awesome !
Thanks a lot

Merged

**Pull request merged**